pull.go

```diff
@@ -45,6 +47,9 @@ type PullOptions struct {
 	// store copies of layer blobs that we pull down, if any.  It should
 	// already exist.
 	BlobDirectory string
+	// AllTags is a value that determines whether or not download all
```

Comment:
Suggest:  AllTags is a boolean value that determines if all tagged images will be downloaded from the repository.  The default is false.